Makefile.core.mk

```diff
@@ -275,10 +275,11 @@ BINARIES:=./istioctl/cmd/istioctl \
   ./cni/cmd/istio-cni \
   ./cni/cmd/istio-cni-repair \
   ./cni/cmd/install-cni \
-  ./tools/istio-iptables
+  ./tools/istio-iptables \
+  ./tools/bug-report
```

Comment:
Strong -1 to introducing a new binary

Reply:
i discussed this at length with @mjog and the reason for it is we need something for older versions of istio that don't have this built in. we still want users to be able to run bug-report binary if it's not part of istioctl. what's the objection to the binary?

Reply:
See comment below (or above, or beside...). Why would a user want a standalone binary? Its already hard enough to manage all the versions of istioctl... if they want it on older versions they should use a new istioctl or we should backport

Reply:
This comment is still valid

Reply:
i'd like to hear from the real @mandarjog - my view on this is it will be more inconvenient for users to download and use a special version of istioctl just for bug-report. not a deal breaker though which is why i'd like to understand better the objections to having a new binary.

Reply:
Its the same, isn't it? Either download bug-report and run it or download old istioctl and run it? Maybe confusing since you have multiple binaries with the same name

Reply:
yeah, it's more the confusion than download problem, although 40Mb is not insignificant. actually to take it one step further i'm thinking about creating a bootstrap script which will test for the arch, grab the path to the latest build and download and run it. that way for all older istio versions we can just point to the script on master.